config/betterquesting/DefaultQuests/Quests/LooktotheEdges-AAAAAAAAAAAAAAAAAAAAGw==/EntTwig-AAAAAAAAAAAAAAAAAAAJIg==.json

```
@@ -8,7 +8,7 @@
   "properties:10": {
     "betterquesting:10": {
       "autoClaim:1": 0,
-      "desc:8": "An Ent Twig will be dropped when an Ent is killed. They are used to make Mutating Sprigs and Mystic Branches. \n\nThey will appear occasionally when trees such as Rowan, Hawthorn and Alder are felled in large quantities.\n\nEnts are rather hard to kill by normal means, but they do have a healthy hatred of certain tools for some reason or another.\n\nTwilight Forest Moss Patch and Biomes \u0027o Plenty moss will drop also.",
+      "desc:8": "An Ent Twig is dropped when an Ent is killed. It is used to craft Mutating Sprigs and Mystic Branches.\n\nThey will appear occasionally when trees such as Rowan, Hawthorn and Alder are felled in large quantities.\n\nEnts are rather hard to kill by normal means, but they do have a healthy hatred of certain tools for some reason or another.\n\nTwilight Forest Moss Patch and Biomes \u0027o Plenty moss will drop also.",
```

This quest doesn't read very well, I made another attempt at it, let me know what you think

```suggestion
      "desc:8": "An Ent Twig is dropped when an Ent is killed, which can be used to craft Mutating Sprigs and Mystic Branches.\n\nEnts will appear occasionally when trees such as Rowan, Hawthorn and Alder are felled in large quantities.\n\nThey are rather hard to kill by normal means, but they do have a healthy hatred of certain tools for some reason or another.\n\nThey can also drop a Twilight Forest Moss Patch or Biomes O\u0027 Plenty Moss.",
```